Description: Don't enable mouse mode by default, it breaks in chroots
Author: Adam Conrad <adconrad@ubuntu.com>
Bug-Ubuntu: https://launchpad.net/bugs/1661691
Last-Update: 2017-03-17

--- vim-8.0.0095.orig/runtime/defaults.vim
+++ vim-8.0.0095/runtime/defaults.vim
@@ -64,12 +64,6 @@ map Q gq
 " Revert with ":iunmap <C-U>".
 inoremap <C-U> <C-G>u<C-U>
 
-" In many terminal emulators the mouse works just fine.  By enabling it you
-" can position the cursor, Visually select and scroll with the mouse.
-if has('mouse')
-  set mouse=a
-endif
-
 " Switch syntax highlighting on when the terminal has colors or when using the
 " GUI (which always has colors).
 if &t_Co > 2 || has("gui_running")
